packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -11,8 +11,14 @@ var _ = require('lodash'),
     isSerializable = require('../lib/is-serializable.js').isSerializable,
     SplitPane = require('ubervu-react-split-pane'),
     localStorageLib = require('../lib/local-storage.js'),
+    CodeMirror = require('react-codemirror'),
     fuzzaldrinPlus = require('fuzzaldrin-plus');
 
+require('codemirror/lib/codemirror.css');
+require('codemirror/addon/fold/foldgutter.css');
+require('codemirror/theme/solarized.css');
```

Comment:
@bogdan-stefan-hs had to move [this](https://github.com/uberVU/react-component-playground/blob/cf52b775fb6feec7de2792c429bf45ca14f87138/src/components/component-playground.jsx#L21-L25) js requires into the `react-code-mirror` fork because of some webpack reason. @bogdan-stefan-hs maybe you can explain what was happening there 💃